### ISSUE TITLE
App crashes caused by Rollbar

### ISSUE THREAD

**Issue by VicZhang1** [rollbar/rollbar-ios#104]
Hi,

I am using Rollbar 1.0.0-alpha11, on Rollbar I find there is a crash caused by Rollbar code as below:
```
NSGenericException: *** Collection <__NSArrayM: 0x604000058510> was mutated while being enumerated.
  CoreFoundation  in __exceptionPreprocess
  libobjc.A.dylib  in objc_exception_throw
  CoreFoundation  in __NSFastEnumerationMutationHandler
  Rollbar  in +[NSJSONSerialization(Rollbar) dataWithJSONObject:options:error:safe:]
  Rollbar  in -[RollbarTelemetry saveTelemetryData]
  Rollbar  in -[RollbarTelemetry recordEventForLevel:type:data:]
  Rollbar  in -[RollbarTelemetry recordConnectivityEventForLevel:status:extraData:]
......
```

Can you take a look at this?

Thanks

**Comment by VicZhang1** [comment]
Thanks @rokob , may I know when you plan to release a new version/patch?

**Comment by VicZhang1** [comment]
Thanks @rokob . 

Is this already released to CocoaPods? I am using Xcode 9.4.1, when I try `pod 'Rollbar', '~> 1.0.0'` and run `pod install`, I get **None of your spec sources contain a spec satisfying the dependency: `Rollbar (~> 1.0.0)`.** message.

It worked for `pod 'Rollbar', '~> 1.0.0alpha11'`.

**Comment by VicZhang1** [comment]
After running `pod repo update`, all is fine now.
